packages/ruleset/src/functions/collection-array-property.js

```diff
@@ -39,8 +39,19 @@ module.exports = function (schema, _opts, context) {
 function collectionArrayProperty(schema, path, apidef) {
   logger.debug(`${ruleId}: checking schema at location: ${path.join('.')}`);
 
-  // Grab the GET operation that defines "schema" as the success response schema.
   const pathString = path[1];
+  if (!apidef.paths[pathString]) {
+    // Spectral has weird behavior where if it finds unexpected characters, like
+    // an apostrophe, it wraps the path segment in quotes. In that case, of course,
+    // the path string won't be in the paths object. Until we better understand the
+    // Spectral behavior, we should just skip this path.
+    logger.error(
+      `${ruleId}: could not find path string ${pathString} in paths object. Skipping check...`
+    );
+    return [];
+  }
```

Comment:
A strange bug I ran into while testing the scoring logic on different APIs. I'll add a test to reproduce before I merge.